mesonbuild/wrap/wraptool.py

```diff
@@ -57,38 +58,30 @@ def add_arguments(parser: 'argparse.ArgumentParser') -> None:
     p.add_argument('project_path')
     p.set_defaults(wrap_func=promote)
 
-def get_result(urlstring: str) -> T.Dict[str, T.Any]:
-    u = open_wrapdburl(urlstring)
-    data = u.read().decode('utf-8')
-    jd = json.loads(data)
-    if jd['output'] != 'ok':
-        print('Got bad output from server.', file=sys.stderr)
-        raise SystemExit(data)
-    assert isinstance(jd, dict)
-    return jd
-
-def get_projectlist() -> T.List[str]:
-    jd = get_result(API_ROOT + 'projects')
-    projects = jd['projects']
-    assert isinstance(projects, list)
-    return projects
+def get_releases() -> T.Dict[str, T.Any]:
```

Comment:
Eventually this should be cached locally for, say, 1 hour.

Reply:
We could do that, yes. I would like to keep it simple for now and get it merged first, we don't have a cache for v1 neither so it's not a regression anyway.

OOC, is there a way in python std to get platform specific user cache dir?

Reply:
[I don't think there is](https://stackoverflow.com/questions/13184414/how-can-i-get-the-path-to-the-appdata-directory-in-python).